docs/general/platform-service-matrix.md

```diff
@@ -31,6 +31,8 @@ The table below lists all the Operating System and Platforms and their supported
 | Xamarin - iOS          |  ✅   |  ✅  |     ✅       |   ✅       |    ✅     |
 | Xamarin - Android      |  ✅   |  ✅  |     ✅       |   ✅       |    ✅     |
 | Xamarin - macOS        |  ❌   |  ❌  |     ✅       |   ✅       |    ✅     |
+| .NET MAUI - Android    |  ❌   |  ❌  |     ✅       |   ✅       |    ✅     |
```

Comment:
We don't want to mention .NET MAUI at this moment, please remove.

Reply:
Done, please review.